tests/conftest.py

```diff
@@ -419,6 +432,15 @@ def augurInitializedSnapshot(fixture, controllerSnapshot):
     return fixture.createSnapshot()
 
 @pytest.fixture(scope="session")
+def augurInitializedWithMocksSnapshot(fixture, augurInitializedSnapshot):
+    fixture.uploadAndAddToController("solidity_test_helpers/Constants.sol")
+    fixture.uploadAllMockContracts()
```

Comment:
We have been, generally, trying to move away from having the tests "upload everything" and more towards a pattern of "use a fixture that just has what you need" in an effort to get localized test runs sped up.  That is, if you just want to run the `test_controlled.py` suite, you don't have to wait for a ton of unrelated things to get uploaded.

The question I have for you regarding this change is, do you find that there are many test fixtures that actually need all of the mocks available, or do most test fixtures just need a couple mocks?  If it is fairly common to need all (or at least the majority) of the mocks available then I think this change is fine.  If most of the time though we only need a few mocks then I would say this change is OK, but be careful to not over-use it.

Reply:
I was noticing that unit tests need all the same mocks, universe, reputation token, market , ... The ones not needed all the time are factory but they are small mock files.